### PR TITLE
Use core tar.gz without packages

### PR DESCRIPTION
**Pull request by fingolfin**
... as it does contain the HTML docs with recent GAP versions.

Caveat: I have not actually tested this PR, as I don't use Gentoo.

### CODE REVIEW

**Comment by kiwifb** [comment]
I tried to do it before and there was stuff missing. I certainly will take a look if you say it is all good now.

**Comment by fingolfin** [comment]
If there is "stuff missing", then we (upstream) would appreciate if there was communication with us to inform us about it, then we can enter a constructive dialog to figure out a way forward :-).

**Comment by kiwifb** [comment]
I certainly will let you know if that happens. 
I guess a somewhat related issue is that not 100% of gap packages have a git repo and therefore no "more or less permanent" mirror for their releases. A small number packages which are on github do not have proper releases either. At least, I can pull a git commit, but it is not good. I kind of got around of all the problematic situation one way or another now, but in the past, the full tarball was the only sure fire way of having some of the packages.

**Comment by kiwifb** [comment]
I am currently manually installing `Makefile.gappkg` under `/usr/share/gap/etc/`. I use it to update/use for any package that has/need it. Is it going away? If not, it would be nice to have it properly installed somewhere.

**Comment by fingolfin** [comment]
> I am currently manually installing `Makefile.gappkg` under `/usr/share/gap/etc/`. I use it to update/use for any package that has/need it.

We considered doing that at some point, but have abandoned this idea, and I would recommend against it. In practice it causes more problems than it solves. I.e., the cases where updating `Makefile.gappkg` by blindly copying a different version over the version in the package is beneficial are almost non-existent -- indeed, we run extensive CI tests on all distributed packages before releasing them, and all builds we make of those packages are using the `Makefile.gappkg` bundled with them, so if there was a problem, we should discover it at this stage.

On the other hand, copying the `Makefile.gappkg` this way can *cause* problems. E.g. some packages have slightly different versions of `Makefile.gappkg` these days with minor patches. Eradicating these differences may or may not cause problems... 

So in a nutshell: just don't do this?

> Is it going away? If not, it would be nice to have it properly installed somewhere.

It is not going away, it is there as a reference for package authors. But we don't install it because there is no need (from our POV) to do so...
